server/src/main/java/org/apache/druid/server/coordinator/rules/PeriodBroadcastDistributionRule.java

```diff
@@ -32,17 +32,21 @@
 public class PeriodBroadcastDistributionRule extends BroadcastDistributionRule
 {
   static final String TYPE = "broadcastByPeriod";
+  public static final boolean DEFAULT_INCLUDE_FUTURE = true;
```

Comment:
Can be private.

Reply:
Oh, it looks that I was wrong. It's being used in `PeriodLoadRuleTest`. Maybe package-private?

Reply:
Ah there're several `DEFAULT_INCLUDE_FUTURE`. Please fix them properly.

Reply:
Sorry, It's my fault. I have never checked before push.